docs/bmquickstart.md

```diff
@@ -141,12 +141,12 @@ Create the installation manifests
 openshift-install create manifests
 ```
 
-Edit the `manifests/cluster-scheduler-02-config.yml` Kubernetes manifest file to prevent Pods from being scheduled on the control plane machines by setting `mastersSchedulable` to `false`.
+Edit the `manifests/cluster-scheduler-02-config.yml` Kubernetes manifest file to prevent Pods from being scheduled on the control plane machines by setting `controlplaneSchedulable` to `false`.
```

Comment:
Is this a spec change that has happened in the latest OpenShift?  I think the spec name is `mastersSchedulable` 

```
spec:
  mastersSchedulable: false
...
```

Reply:
@whowutwut oversight on my part. Reverted back to `mastersSchedulable`.